packages/backend/src/ee/database/entities/aiArtifacts.ts

```diff
@@ -0,0 +1,42 @@
+import { Knex } from 'knex';
+
+export const AiArtifactsTableName = 'ai_artifacts';
+
+export type DbAiArtifact = {
+    ai_artifact_uuid: string;
+    created_at: Date;
+    ai_thread_uuid: string;
+    artifact_type: string;
+    saved_query_uuid: string | null;
```

Comment:
should we move this one to the artifact version? I guess it makes sense to allow users to save charts from different versions of the artifact 🤔